flake.lock

```diff
@@ -366,7 +484,7 @@
     },
     "sops-nix": {
       "inputs": {
-        "nixpkgs": "nixpkgs_5",
+        "nixpkgs": "nixpkgs_6",
```

Comment:
do you mind garbage-collecting those instances of nixpkgs a bit?

Reply:
I've added one nixpkgs and Nix renumbered them.

I guess `follows` might be an option, if that's what you mean? But I don't think you'll be quite happy with that either, because I can't really promise hercules-ci-agent to be quite as portable between nixpkgs versions as you may hope, especially because the `nix` package does not have C++ api stability.

Reply:
Ok, forget it then